src/test/java/org/opensearch/ad/transport/handler/AbstractIndexHandlerTest.java

```diff
@@ -73,6 +73,9 @@ enum IndexCreation {
     @Mock
     protected IndexNameExpressionResolver indexNameResolver;
 
+    @Mock
+    protected static OpenSearchAsyncClient sdkJavaAsyncClient;
```

Comment:
You are mocking it already below. No need to mock here. Just move this after line 60 and say:
```
protected  OpenSearchAsyncClient sdkJavaAsyncClient;